docs/specification/profiles/ros1.md

```diff
@@ -1,8 +1,19 @@
-## ROS1 Profile
+## ROS1
 
-Profile name: "ros1".
+Profile: **ros1**
 
-Channel info records serialized with the "ros1" profile may include the following fields:
+The `ros1` profile describes how to create mcap files for v1 of the [ROS](https://ros.org/) framework.
+
+### Channel Info
+
+| field       | value            |
+| ----------- | ---------------- |
+| encoding    | ros1             |
+| schema      | .MSG             |
+| schema_name | package/datatype |
```

Comment:
```suggestion
| encoding    | [ROS 1 binary serialization](http://wiki.ros.org/msg)             |
| schema      | [.msg](http://wiki.ros.org/msg)             |
| schema_name | `package/datatype`, e.g. `my_msgs/Thing` |
```

Reply:
You want the encoding string to be `ROS 1 binary serialization` this seems rather verbose...

Reply:
These strings should also match https://github.com/foxglove/mcap/pull/60

Reply:
Ok, I misunderstood this table because it's not clear if this column has exact values that must be used or just descriptions. If it is a literal value you're supposed to put in the encoding field then it should be a code snippet such as `ros1`